### ISSUE TITLE
Front-end highlighter redesign: Missing screen reader announcements for "Rescan This Page" button

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1628]
When the "Rescan This Page" button is used in the front-end highlighter, there are no screen reader announcments as to whether the scan was successful.

How to fix
Add a hidden screen-reader only div with role="status" aria-live="polite" aria-atomic="true" and insert messages into it as needed to communicate scan status.
